packages/passport/sdk/src/config/config.ts

```diff
@@ -96,6 +96,7 @@ export class PassportConfiguration {
         }),
         passport: createConfig({
           basePath: overrides.passportMrBasePath,
+          headers: overrides.headers,
```

Comment:
This can be dangerous because it is a potential leak for our SDK and risks all the API calls.

Reply:
It's part of the overrides that already allow for custom base paths, relayer URLs, magic keys and more, all of which are accessible for anyone using the SDK. Additionally, there's nothing stopping an attacker from just forking our code and manually adding whatever headers they want anyway, then using that forked code in their application. I don't see this as introducing any risks beyond what we already have as part of our codebase or architecture.